edsdme/scripts/utils.js

```diff
@@ -341,7 +327,8 @@ function extractTableCollectionTags(el) {
     if (rowTitle === 'collection-tags') {
       const [collectionTagsEl] = colsContent;
       const collectionTags = Array.from(collectionTagsEl.querySelectorAll('li'), (li) => `"${li.textContent.trim().toLowerCase()}"`);
-      tableCollectionTags = [...tableCollectionTags, ...collectionTags];
+      const filtered = collectionTags.filter((tag) => !tag.includes('caas:adobe-partners/cpp/region/'));
```

Comment:
@cod17828 sorry if this was not clear, but I don't think we should have any code to handle region tags separately from other tags